clients/js/src/embeddings/DefaultEmbeddingFunction.ts

```diff
@@ -85,12 +86,28 @@ export class DefaultEmbeddingFunction implements IEmbeddingFunction {
 
   /** @ignore */
   static async import(): Promise<{
-    // @ts-ignore
+    // @ts-expect-error
     pipeline: typeof import("chromadb-default-embed");
   }> {
     try {
-      // @ts-ignore
-      const { pipeline } = await import("chromadb-default-embed");
+      let importResult;
+      if (isBrowser()) {
+        importResult = await import(
+          // todo: we can't import chromadb-default-embed here yet because the `build` script was not run before publishing our fork to NPM, so the entrypoint in our forked package points to a non-existent file.
+          // @ts-expect-error
+          "https://unpkg.com/@xenova/transformers@2.13.2"
```

Comment:
our fork is published under Jeff's account on npm, so we may have to want for him to get back to resolve this
happy to wait until this is resolved before merging

Reply:
we could also wait to resolve this until the next version bump on our fork, since fixing now would mean decoupling our fork version # with upstream (or I guess we could publish as 2.13.2-built or something)

Reply:
created an issue and a reminder to follow up with Jeff next week
https://github.com/chroma-core/chroma/issues/2055